contributor-docs/releasing.md

```diff
@@ -0,0 +1,57 @@
+
+We use [semver](https://semver.org/) for our release naming but we’re currently pre-v1 so every change is a patch release, and may contain breaking changes.
+
+It's preferable to group changes so we don't release a new version for every small change, but for fixes or urgent features it's okay to make releases for a small amount of changes.
+
+## Release process
+
+### Authentication
+
+Before publishing you need to authenticate with [npm](https://www.npmjs.com/) and [RubyGems](https://rubygems.org/).
+
+For [RubyGems](https://rubygems.org/) create an account with your personal email and set up two factor authentication. Ask an owner of the [`primer_view_components` gem](https://rubygems.org/gems/primer_view_components) to add you as a contributor, and then run `gem signin` to authenticate with the CLI.
+
+For [npm](https://www.npmjs.com/) create an account with your personal email and set up two factor authentication. Ask one of the collaborators on the [`@primer/view-components` package](https://www.npmjs.com/package/@primer/view-components) to add you to the Primer team, then run `npm adduser` to authenticate with the CLI. You can confirm your authentication with `npm whoami`.
+
+### Preparing a release
+
+Run `script/release` to prepare a release. This will check you're on the latest version of the `main` branch, bump the gem and npm package versions, tag the Changelog, and open a pull request with the changes. Even if no changes have been made to the JavaScript package we make a release anyway so the version numbers are always in sync. Once the tests have passed you can merge this pull request.
+
+### Publishing a release
+
+When the release pull request is merged into `main`, pull the changes and run `script/publish` to publish the library on rubygems and npm.
+
+```ruby
+script/publish
+npm publish --access public
+```
+
+This will also open a pre-filled release on GitHub, if you’re happy with the release notes hit ‘Publish’.
+
+## Revert plan
+
+If you’ve released a version of the library that introduces significant visual regressions or unintended breaking changes the best thing to do is make a new release which reverts the offending changes. Do not include any other unreleased changes besides the reverts, as they could have unintended side-effects.
+
+[Process TBC]
+
+## Docs deployment
+
+The documentation site is deployed to [primer.style/view-components](https://primer.style/view-components/) by a Vercel integration, it is automatically deployed with changes to `main`.
+
+## Storybook deployment
+
+Storybook needs to be manually deployed to Heroku using chatops. There are two environments:
+
+- `production` at [primer-view-components.herokuapp.com](https://primer-view-components.herokuapp.com/view-components/stories), this can be accessed from [primer.style/view-components/stories](https://primer.style/view-components/stories/)
+- `staging` at [?]
```

Comment:
I can't find any mention of this in the ops channel but it's listed as an option on the Contributing page, any ideas @joelhawksley?

Reply:
I think we can forget about staging, I don't think it's ever used